go/host/rpc/clientapi/client_api_eth.go

```diff
@@ -48,6 +49,33 @@ func (api *EthereumAPI) BlockNumber() hexutil.Uint64 {
 	return hexutil.Uint64(header.Number.Uint64())
 }
 
+/*Access list object with the following fields:
+
+accessList: A list of objects with the following fields:
+address: Addresses to be accessed by the transaction.
+storageKeys: Storage keys to be accessed by the transaction.
+gasUsed: A hexadecimal string representing the approximate gas cost for the transaction if the access list is included.*/
+
+type AccessListObject struct {
```

Comment:
What's this thing for? Not sure if you meant it for this PR but be good if the comment above explained what the access list RPC endpoint would be used for. Maybe we've talked about it and I've forgotten.